front/types/assistant/conversation.ts

```diff
@@ -126,6 +126,7 @@ export type ConversationType = {
   title: string | null;
   visibility: ConversationVisibility;
   content: (UserMessageType[] | AgentMessageType[])[];
+  participants: { id: string; name: string; pictureUrl: string | null }[];
```

Comment:
`id` is unclear and a client-side concern

I think we should have a UserParticipant and an AgentParticipant with specific fields I guess configurationId handle and pictureUrl. User can have specific fields about the user as appropriate

Then from there you'll be able to set a unique key client side

Reply:
I added a commit!